README.md

```diff
@@ -43,35 +44,6 @@ following:
 *   _clang-format 5.0.1+_ (ships as part of _clang_)
 
 
-### Mac OS
-
-For Ginkgo core library:
-
-*   _cmake 3.1+_
-*   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_ (__TODO__: verify)
-    *   _clang 3.3+_ (__TODO__: verify)
-    *   _Apple LLVM 8.0+_ (__TODO__: verify)
-
-The Ginkgo CUDA module has the following __additional__ requirements:
-
-*   _cmake 3.8+_
-*   _CUDA 7.0+_ (__TODO__: verify)
-*   Any host compiler restrictions your version of CUDA may impose also apply
-    here. For the newest CUDA version, this information can be found in the
-    [CUDA installation guide](http://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html)
-
-In addition, if you want to contribute code to Ginkgo, you will also need the
-following:
-
-*   _clang-format 5.0.1+_ (ships as part of _clang_)
```

Comment:
Was this deleted by accident?

Reply:
That was the Mac OS section, which was exactly the same as the Linux one except for the Apple LLVM compiler. In addition, many linux specific things apply to mac (they can use a real gcc). So I merged the two sections. That way we should also make less mistakes in keeping all the fields up to date.